tox.ini

```
@@ -0,0 +1,36 @@
+[tox]
+skipsdist=True
+envlist=pycodestyle,pylint,yamllint
+
+[test-deps]
+deps =
+  -cconstraints.txt
+  -raws/requirements.txt
+
+[test-deps:pylint]
```

This should remain 
```suggestion
[testenv:pylint]
```
For tox to notice it.